### PR TITLE
Add Weights and Biases reference

### PR DESCRIPTION
**Pull request by AyushExel**
Weights & Biases helps turn deep learning research projects into deployed software by helping teams track their models, visualize model performance and easily automate training and improving models.

### CODE REVIEW

**Comment by axsaucedo** [comment]
Nice one

**Comment by axsaucedo** [comment]
Actually, their product seems to be primarily commercial, so it will need to be in the commercial section

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Studio.ML](https://github.com/studioml/studio) ![](https://img.shields.io/github/stars/studioml/studio.svg?style=social) - Model management framework which minimizes the overhead involved with scheduling, running, monitoring and managing artifacts of your machine learning experiments.
 * [TerminusDB](https://terminusdb.com/) ![](https://img.shields.io/github/stars/terminusdb/terminusdb.svg?style=social) - A graph database management system that stores data like git.
 * [TRAINS](https://github.com/allegroai/trains) ![](https://img.shields.io/github/stars/allegroai/trains.svg?style=social) - Auto-Magical Experiment Manager & Version Control for AI.
+* [Weights & Biases](https://wandb.ai/site) ![](https://img.shields.io/github/stars/wandb/client?style=social)- Machine learning experiment tracking, dataset versioning, hyperparameter search, visualization, and collaboration
```
Comment (axsaucedo) [inline README.md:144]: move to commercial section

**Review by axsaucedo — CHANGES_REQUESTED** [review]

**Comment by axsaucedo** [comment]
@AyushExel will you be able to add the requestsed changes?

**Comment by AyushExel** [comment]
@axsaucedo W&B is free for individual users and academics. So I don't think it should be kept under the commercial section. Are you sure about that?

**Comment by axsaucedo** [comment]
@AyushExel yep, commercial tools in the commercial section

**Comment by AyushExel** [comment]
Done

**Comment by axsaucedo** [comment]
@AyushExel awesome - thanks!

**Pull request merged**